src/data/default.js

```diff
@@ -23,16 +23,18 @@ data.glow_iv = config.map.glow.iv;
 // Default filter options for new users/cache clears
 data.default_show_pokemon = config.map.filters.pokemon;
 data.default_show_raids = config.map.filters.raids;
+data.default_show_raidTimers = config.map.filters.raidTimers;
```

Comment:
Hmm maybe we should just use underscores instead of camel case for the rest of it too tbh.
```suggestion
data.default_show_raid_timers = config.map.filters.raidTimers;
```

Reply:
Done.